sdk/instrumentation/bodyattribute/bodyattribute.go

```diff
@@ -3,10 +3,14 @@ package bodyattribute // import "github.com/hypertrace/goagent/sdk/instrumentati
 import (
 	"encoding/base64"
 	"fmt"
+	"strings"
+	"unicode/utf8"
 
 	"github.com/hypertrace/goagent/sdk"
 )
 
+const utf8Replacement = "�"
```

Comment:
Also, I am not sure if substituting this sequence of bytes is sufficient to fix these errors.

Reply:
> I am not sure if substituting this sequence of bytes is sufficient to fix these errors.

Hmm, it should resolve any errors from invalid bytes being in the request or response body from the protobuf perspective? (at least I see this sample export successfully now)
Or are you thinking there are other cases where the replace wouldn't actually replace all invalid byte sequence?

If they appear in headers we'd still run into exception
(but not sure how likely that is for us to be concerned of right now?

Reply:
I am trying to understand what causes this invalid byte sequence.